cmd/ttn-lw-cli/commands/gateways.go

```diff
@@ -434,41 +435,76 @@ var (
 	gatewaysConnectionStats = &cobra.Command{
 		Use:     "get-connection-stats [gateway-id]",
 		Aliases: []string{"connection-stats", "cnx-stats", "stats"},
-		Short:   "Get connection stats for a gateway",
+		Short:   "Get connection stats for a (group of) gateway(s).",
+		Long: `Get connection stats for a (group of) gateway(s).
+		Use the --gateway-ids flag for multiple gateways.
+		If both the parameter and the flag are provided, the parameter is ignored.`,
 		RunE: func(cmd *cobra.Command, args []string) error {
-			gtwID, err := getGatewayID(cmd.Flags(), args, true)
+			ids, err := cmd.Flags().GetStringSlice("gateway-ids")
 			if err != nil {
 				return err
 			}
 
-			is, err := api.Dial(ctx, config.IdentityServerGRPCAddress)
+			gtwID, err := getGatewayID(cmd.Flags(), args, true)
 			if err != nil {
-				return err
+				if !errors.IsInvalidArgument(err) || len(ids) == 0 {
+					return err
+				}
 			}
```

Comment:
I think it's better to solve this as follows:

- Define the new `gateway-ids` flag
- Replace the existing `gateway-id` flag with an alias to the `gateway-ids` flag
- Also support multiple IDs in arguments